.github/workflows/label_prs.yaml

```diff
@@ -39,20 +41,19 @@ jobs:
               - 'MODULE.*'
               - 'WORKSPACE'
               - 'bazel/**'
-              - 'common/**'
               - 'github_tools/**'
-              - 'installers/**'
-              - 'language_server/**'
               - 'proposal/scripts/**'
               - 'scripts/**'
-              - 'testing/**'
-              - 'third_party/**'
-              - 'utils/**'
             proposal:
```

Comment:
Please call out why this is using "added"

Reply:
Added comments.